src/utils/__tests__/addDependenciesToPackages.test.js

```diff
@@ -137,9 +137,27 @@ describe('utils/addDependenciesToPackages', () => {
     );
   });
 
+  test('should be able to add packages with tagged versions (without specifying)', async () => {
```

Comment:
This would have always worked, but decided to add a defensive test just in case.